### ISSUE TITLE
Where are the past releases? + PyPI release is out-of-date

### ISSUE THREAD

**Issue by nchammas** [datamade/probablepeople#39]
It seems strange there is [only one release](https://github.com/datamade/probablepeople/releases), tagged v0.5.3. Are there no releases prior to that? Why aren't they tagged in git?

On a related note, why is the [latest release shown on PyPI](https://pypi.org/project/probablepeople/) at version 0.5.2?

**Comment by fgregg** [comment]
It seems like you have expectations about how we use git, github, and pypi that do not match our practices.

**Comment by nchammas** [comment]
Right, and that's totally fine. I was just trying to understand why things are laid out the way they are, and whether it was by design, since it doesn't match the conventions I'm familiar with (which I think are common conventions, but perhaps I am mistaken about that).

It sounds like it _is_ by design and, judging from the quick close, I should not expect any elaboration. Or perhaps this is already documented somewhere and I missed it.

Either way, apologies for the noise.

**Comment by fgregg** [comment]
Design is definitely putting it too strongly. Is there something in particular you are trying to do that you can't?

**Comment by nchammas** [comment]
Nope, not at the moment, since I don't use probablepeople. But if I were using it in my projects, I could imagine the following scenarios coming up at some point:

1. Hmm, I think I'm hitting a bug in version X of probablepeople. Can I easily find the code the corresponds to version X so I can investigate further?
2. I noticed a change in behavior in my application sometime after I upgraded from version X of probablepeople to version Y. I'm not sure if my project is culpable, or if it's probablepeople. Can I easily diff the two versions of probablepeople to see if that yields any clues?
3. A known bug in probablepeople got fixed recently. Did the fix get released so I can upgrade my dependency accordingly? What version do I need to upgrade to?

In projects following common git/GitHub conventions (at least, from what I know), all these scenarios would be easy to navigate. Each release would be tagged in git, which means the version is navigable and diffable against other versions in GitHub. And each tagged release in git would have a corresponding release on PyPI.

Right now there is no way for people to understand probablepeople's behavior at arbitrary releases. Generally this is fine, since most people aren't going to need to do that. This kind of stuff does come up though when people need to track issues with their upstream dependencies, or when as a maintainer you are trying to correlate user reports with code changes.

It'll also be confusing to people that the tagged release on GitHub (0.5.3) is newer than the latest release on PyPI (0.5.2). I would wager that most would assume, like I did, that that is a mistake. Why would something be tagged with a release version but not be published to PyPI? There can perhaps be a good reason for that, but it won't be obvious to the typical open source user or contributor.

I hope this clarifies what prompted me to open this issue.
